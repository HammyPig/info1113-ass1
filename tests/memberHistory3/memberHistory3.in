ADD MEMBER johnCoolMan3000
ADD member you, the marker
add member (thats right you heard me)
ADD COLLECTION books.csv
MEMBER HISTORY 100001
RENT 100000 111111
Member HIsTORY 100000
RELINQUISH 100000 111111
member history 100000
RELINQUISH 100000 111111
MEMBER HISTORY 100000
RENT 100000 111111
RELINQUISH 100000 111111
RENT 100000 111111
RELINQUISH 100000 111111
RENT 100000 111111
RELINQUISH 100000 111111
MEMBER HISTORY 100000
RENT 100001 111111
RELINQUISH 100001 111111
RENT 100001 111112
RELINQUISH 100001 111112
RENT 100001 111113
RELINQUISH 100001 111113
MEMBER HISTORY 100001
RENT 100002 111113
RELINQUISH 100002 111113
MEMBER HISTORY 100002
MEMBER HISTORY 999999