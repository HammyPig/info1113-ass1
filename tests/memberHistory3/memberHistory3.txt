recognises when no members exist, deals with complex interactions (duplicate relinquishing and multiple members)