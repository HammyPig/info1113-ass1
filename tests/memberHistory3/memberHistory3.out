user: Success.

user: Success.

user: Success.

user: 17 books successfully added.

user: No rental history for member.

user: Success.

user: No rental history for member.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Unable to return book.

user: To Kill a Mockingbird (Harper Lee)

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)

user: Success.

user: Success.

user: The Iliad (Homer)

user: No such member in system.

user: Ending Library process.
